huggingface/pytorch/inference/docker/1.10/py3/sdk1.19.1/Dockerfile.neuron

```diff
@@ -0,0 +1,139 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.1.8
```

Comment:
Any specific reason for using MMS and not torchserve. Rest of the DLC's are using torchserve